messageBundles/gpii-app-messageBundle_bg.json

```diff
@@ -27,7 +27,12 @@
 
     "gpii_psp_errorDialog_titlebarAppName": "GPII Грешка",
 
-    "gpii_psp_aboutDialog_titlebarAppName": "Относно GPII Auto-personalization",
+    "gpii_psp_aboutDialog_titlebarAppName": "Относно GPII Авто-персонализиране",
+    "gpii_psp_aboutDialog_title": "GPII Авто-персонализиране",
+    "gpii_psp_aboutDialog_titlebarAppName": "Относно GPII Авто-персонализиране",
+    "gpii_psp_aboutDialog_update": "Провери за ъпдейти",
+    "gpii_psp_aboutDialog_userListenersText": "Средства за вписване за тази машина: %items",
+    "gpii_psp_aboutDialog_version": "версия %item",
```

Comment:
I do not think that `%items` and `%item` are good placeholder names. Why not use `%listeners` and `%version`?